UnityProject/Assets/Scripts/Objects/Machines/QuantumPad.cs

```diff
@@ -184,6 +187,14 @@ public void ServerDetectObjectsOnTile()
 			{
 				TransportUtility.TransportObjectAndPulled(item, travelCoord);
 				somethingTeleported = true;
+
+				if (item.TryGetComponent<Bread>(out _) && DMMath.Prob(25))
+				{
+					Spawn.ServerPrefab(MutatedBread, travelCoord);
+					Destroy(item);
```

Comment:
```suggestion
					Despawn.ServerSingle(item);
```
Similar to `Spawn.ServerPrefab()` we have `Despawn.ServerSingle()`. These are networked and pooled.